pom.xml

```diff
@@ -356,6 +356,7 @@
                   <manifestEntries>
                     <Main-Class>org.folio.rest.RestLauncher</Main-Class>
                     <Main-Verticle>org.folio.rest.RestVerticle</Main-Verticle>
+                    <Multi-Release>true</Multi-Release>
```

Comment:
Please also add this to the maven-shade-plugin configuration:

https://github.com/folio-org/okapi/pull/964/commits/f4b5b33dfed59e2d64340a9a18a5cc479957afcf

Reply:
Without it, we get some disturbing messages when we start with the fat jar:

```
adam@nzxt:~/folio/mod-user-import$ java -jar target/mod-user-import-fat.jar 
ERROR StatusLogger Unrecognized format specifier [d]
ERROR StatusLogger Unrecognized conversion specifier [d] starting at position 16 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [thread]
ERROR StatusLogger Unrecognized conversion specifier [thread] starting at position 25 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [level]
ERROR StatusLogger Unrecognized conversion specifier [level] starting at position 35 in conversion pattern.
ERROR StatusLogger Unrecognized format specifier [logger]
```

Reply:
done